queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '01-01-2016' AND '01-01-2019';
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT * FROM animals WHERE name = 'Agumon' OR  name = 'Pikachu';
```

Comment:
- [ ] Kindly add a query that lists the name and escape attempts of animals that weigh more than 10.5kg per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=List%20name%20and%20escape%20attempts%20of%20animals%20that%20weigh%20more%20than%2010.5kg).